index.html

```diff
@@ -63,6 +63,224 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ol>
+            </div>
+            <div class="flex-item-width paragraph">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="flex-item-width buttons-flix">
+                <ul class="ul-list">
+                    <button class="buttons"> <li>HTML</li></button>
+                    <button class="buttons"><li>CSS</li></button>
+                    <button class="buttons">
+                <li>JavaScript</li></button>
+                </ul>
+            </div>
+            <br>
+            <div class="flex-item-width">
+                <button id="seeProject-button">See Project</button>
+            </div>
+        </div>
+        <!-- Work section item four -->
+        <div class="works-grid-item">
+            <div class="template-img flex-item-width">
+                <img src="img/Snapshoot Portfolio 2.png" alt="">
+            </div>
+            <div class="Tonic flex-item-width">
+                <h2>Multi-Post Stories
+                </h2>
+            </div>
+            <div class="ol-list flex-item-width">
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ol>
+            </div>
+            <div class="flex-item-width paragraph">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="flex-item-width buttons-flix">
+                <ul class="ul-list">
+                    <button class="buttons"> <li>HTML</li></button>
+                    <button class="buttons"><li>CSS</li></button>
+                    <button class="buttons">
+            <li>JavaScript</li></button>
+                </ul>
+            </div>
+            <br>
+            <div class="flex-item-width">
+                <button id="seeProject-button">See Project</button>
+            </div>
+        </div>
+        <br><br>
+    </section>
+
+    <!-- About Myself Section -->
+    <section class="myself-container">
+
+        <div class="myself-flex-items flex-item-width heading2">
+            <h2>About Myself</h2>
+        </div>
+        <div class="myself-flex-items flex-item-width">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <br>
+        <div class="myself-flex-items flex-item-width lets-connect">
+            <a href="#">LET'S CONNECT</a>
```

Comment:
- [ ] This section is not a link, as such, should be treated as a header with hx tag 👍